bitnami/spring-cloud-dataflow/README.md

```diff
@@ -242,6 +242,7 @@ The following tables lists the configurable parameters of the Spring Cloud Data
 | `mariadb.db.user`                            | Username of new user to create                                                                         | `dataflow`                                              |
 | `mariadb.db.password`                        | Password for the new user                                                                              | `change-me`_                                            |
 | `mariadb.initdbScripts`                      | Dictionary of initdb scripts                                                                           | Check `values.yaml` file                                |
+| `externalDatabase.url`                       | URL for the external database. Overrides external scheme, host, port, database, and jdbc parameters.   | `""`                                                    |
```

Comment:
Just a small petition. I just saw that `externalDatabase.driver` and `externalDatabase.scheme` are not documented in the README. Could you please add them?

Reply:
Good suggestion, done.